services/report/__init__.py

```diff
@@ -303,10 +282,12 @@ def get_existing_report_for_commit(
         files = commit.report_json["files"]
         sessions = commit.report_json["sessions"]
         totals = commit.totals
-        res = self.build_report(
-            chunks, files, sessions, totals, report_class=report_class
+
+        if report_class is None:
+            report_class = Report
```

Comment:
can this be anything else now? Other than `Report` I mean.

Reply:
if it can't be anything else then `report_class` is looking obsolete, but I understand we don't want to mess with the function interface just yet

Reply:
There is still the `ReadOnlyReport` that some callers are passing explicitly. So until I can get rid of that, I will keep this as is.